test/proposals/proposals.jl

```diff
@@ -32,3 +33,8 @@ end
 
 @testset "RWalk" begin
 end
+
+@testset "RStagger" begin
+    # printing
+    @test sprint(show, Proposals.RStagger(;ratio=0.5, walks=25, scale=1)) == "NestedSamplers.Proposals.RStagger"
```

Comment:
There's a better way to do this:

```suggestion
    @test Proposals.RStagger(;ratio=0.5, walks=25, scale=1) isa NestedSamplers.Proposals.RStagger
```